psi4/src/psi4/libmints/writer.cc

```diff
@@ -1417,6 +1417,36 @@ void MOWriter::write_mos(Molecule & mol){
 
     CharacterTable ct = mol.point_group()->char_table();
 
+    BasisSet& basisset = *wavefunction_->basisset().get();
+    
+    std::vector<std::string> ao_labels;
+    for (int s = 0; s < basisset.nshell(); s++) {
+        GaussianShell shell = basisset.shell(s);
+        int center = shell.ncenter()+1;
+        int am = shell.am();
+        char amchar = shell.amchar();
+        std::string basename = mol.symbol(shell.ncenter()) + std::to_string(center) + " ";
+        basename += char(amchar);
+        if (amchar == 'd' && (basisset.has_puream())) {
+            for (int i = 1; i < 6; i++) ao_labels.push_back(basename+" "+std::to_string(i));
```

Comment:
@amjames I agree that the space needs to be removed. After @jturney suggested explicitly labeling the AO's with '+1', '-1', etc. I have simply removed the space because the underscore might reduce readability with the plus and minus signs right next to it. Any thoughts on this?